static/app/views/dashboards/widgetBuilder/components/typeSelector.tsx

```diff
@@ -60,6 +60,13 @@ function WidgetBuilderTypeSelector() {
               payload: [state.query[0]],
             });
           }
+
+          if (newValue.value === DisplayType.BIG_NUMBER) {
+            dispatch({
+              type: BuilderStateAction.SET_SORT,
+              payload: [],
+            });
+          }
```

Comment:
Can we actually put this in the hook? Basically in the `dispatch` call, find out where we're setting the display type and you can basically add this there to wipe out the payload. It'll be our first field interaction and I'd like to see how it works. Hopefully it doesn't blow up lol

Reply:
LOLLL it thankfully does not blow anything up, I'll push this change